src/components/newsArchive.scss

```diff
@@ -0,0 +1,15 @@
+.news-archive-container {
+  margin-top: 130px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 18px 80px;
+}
+
+
+@media only screen and (max-width: 375px) {
```

Comment:
I'm guessing this width value is from the production site? It seems too small to me, and the news items look really bad from 376px until ~ 900px or so.

I think what I would do is increase that breakpoint from 375 to something a little larger, and then add a second breakpoint between that value and 900px (or thereabouts) to display items in 2 columns instead of 3, and finally go to 3 columns for larger screens.

Reply:
I'll play with this a little bit and get things looking a little better in between those breakpoints you mentioned!